go.mod

```diff
@@ -41,4 +42,6 @@ replace (
 	github.com/docker/docker => github.com/docker/docker v1.4.2-0.20200221181110-62bd5a33f707
 	// Darwin cross-build required version
 	golang.org/x/sys => golang.org/x/sys v0.0.0-20200302150141-5c8b2ff67527
+
+	knative.dev/client-contrib/plugins/source-kafka => github.com/dsimansk/kn-source-kafka v0.0.0-20201022153314-e9a483f1b41c
```

Comment:
That's my fork with slight modification for now. https://github.com/dsimansk/kn-source-kafka/commit/e9a483f1b41c2fcd609726d4bc31d1f2232c710e